CogniteSdk.Types/Alpha/Simulators/SimulatorValue.cs

```diff
@@ -114,5 +136,83 @@ public override bool Equals(object obj)
             /// </summary>
             public override int GetHashCode() => Value.GetHashCode();
         }
+
+        /// <summary>
+        /// Simulator value that contains a list of string values.
+        /// </summary>
+        public sealed class StringArray : SimulatorValue
+        {
+            /// <summary>
+            /// Initialize new string list value.
+            /// </summary>
+            /// <param name="value">The value to set.</param>
+            public StringArray(IEnumerable<string> value)
+            {
+                Type = SimulatorValueType.STRING_ARRAY;
+                Value = value;
+            }
+
+            /// <summary>
+            /// The contained string array value.
+            /// </summary>
+            public IEnumerable<string> Value { get; set; }
+
+            /// <summary>
+            /// Return string representation of the string array value, i.e the string array itself.
+            /// </summary>
+            public override string ToString() => string.Join(", ", Value);
+
+            /// <summary>
+            /// Check if two string array values are equal.
+            /// </summary>
+            public override bool Equals(object obj)
+            {
+                return obj != null && Value.SequenceEqual((obj as StringArray).Value);
+            }
+
+            /// <summary>
```

Comment:
Value.GetHashCode() will not provide a correct hash code for a collection.

`public override int GetHashCode() 
	{
		HashCode hash = new HashCode();
		foreach (var str in Value)
		{
			hash.Add(str);
		}
		return hash.ToHashCode();
	}`

Reply:
Since there is no HashCode in dotnet 2. We could simply use something like this:

```
public override int GetHashCode()
{
    unchecked
    {
        int hash = 17;
        const int multiplier = 23;

        foreach (var item in Value)
        {
            hash = hash * multiplier + (item?.GetHashCode() ?? 0);
        }

        return hash;
    }
}
```
prime numbers 17 and 23 are chosen based on common practices and recommendations for implementing hash code functions https://stackoverflow.com/questions/263400/what-is-the-best-algorithm-for-overriding-gethashcode